core/field.ts

```diff
@@ -1401,6 +1435,30 @@ export abstract class Field<T = any>
     }
   }
 
+  /** See IFocusableNode.getFocusableElement. */
+  getFocusableElement(): HTMLElement | SVGElement {
+    if (!this.fieldGroup_) {
+      throw Error('This field currently has no representative DOM element.');
+    }
+    return this.fieldGroup_;
+  }
+
+  /** See IFocusableNode.getFocusableTree. */
+  getFocusableTree(): IFocusableTree {
+    const block = this.getSourceBlock();
+    if (!block) {
+      throw new UnattachedFieldError();
+    }
+    // TODO: Remove the latter two casts once WorkspaceSvg is a focusable tree.
```

Comment:
Reminder to fix this & update workspace to integrate with fields correctly once #8916 is merged.

Reply:
Since this PR is now chained off of the other branches, this has been fixed.